package.json

```diff
@@ -97,6 +97,16 @@
           "markdownDescription": "If not set, will inherit from the `no_proxy` or `NO_PROXY` environment variables. `http.proxySupport` must be set to `on` or `off`, otherwise VS Code will override the proxy agent set by the plugin.",
           "type": "string",
           "default": ""
+        },
+        "coder.defaultUrl": {
```

Comment:
Could you add a note about auto login precedence/falling back to CODER_URL to both of these descriptions?

Reply:
Will do for `defaultUrl`, the other already has it though, is there something specific you had in mind?  Or do you mean the `or` is not clear on precedence, and it would be nice to be explicit about it?

Reply:
Although, I am tempted to remove the autologin details from `defaultUrl` because it feels like a distraction, all that information should maybe just be on the `autologin` setting.  What do you think?

Reply:
Yeah I meant the or wasn't quite explicit, but I agree on removing it altogether. I would call it the "autologin_url" or something in the description instead of referring to the environment var specifically, so they go check that settings description and find the precedence logic.

Reply:
It also is used to populate the default selection in the URL selector, so I was hesitant to call it specifically something related only to autologin, but there might be a better name somewhere that encapsulates both things.